dist/selectivity-full.js

```diff
@@ -3567,10 +3573,17 @@ var callSuper = Selectivity.inherits(MultipleSelectivity, {
 
     _renderSelectedItem: function(item) {
 
-        this.$searchInput.before(this.template('multipleSelectedItem', $.extend({
+        var options = $.extend({
             highlighted: (item.id === this._highlightedItemId),
             removable: !this.options.readOnly
-        }, item)));
+        }, item);
+
+        this.$searchInput.before(this.template('multipleSelectedItem', options));
```

Comment:
I think you can leave this as it was before without assigning the options variable, and just passing `item` to the `selectOptionCompliance` template.